src/components/GemBlock.tsx

```diff
@@ -13,7 +13,7 @@ import classnames, {
 } from 'classnames/tailwind'
 import useBreakpoints from 'hooks/useBreakpoints'
 
-const gemCard = space('-space-y-44')
+const gemCard = space('-space-y-48')
```

Comment:
Fixed clipped `ArcText` and block positioning for it (we used `scale`, so it got broken, explained it more in `GemWithArc.tsx`)